nerder/gh-explorer
# #11: implement ResultsPanel (closes #11)

Issue #11
## Test Plan
- Open the app and you will see a presentation page, inside of a floating `Panel`
- Search for reactjs and you will see some results as output (reactjs is hardcoded for testing purpose only)
- On results page scroll down and click the "go to top" button, you will see the animation
- Search for any other string and you will get a 404 not found page
